lib/pages/chat/chat.dart

```diff
@@ -1770,6 +1779,121 @@ class ChatController extends State<Chat>
     }
   }
 
+  List<PopupMenuEntry<ChatAppBarActions>> appBarActionsBuilder() {
+    final listAction = [
+      if (PlatformInfos.isAndroid) ...[
+        if (selectedEvents.every((event) => event.isVideoOrImage))
+          ChatAppBarActions.saveToGallery,
+        if (selectedEvents
+            .every((event) => event.hasAttachment && !event.isVideoOrImage))
+          ChatAppBarActions.saveToDownload,
```

Comment:
IMO, only show these actions if `selectedEvents`'s length is 1

Reply:
and please don't show `save to gallery` action without logic

Reply:
Already did that, you can try